client/my-sites/plan-features-2023-grid/components/features.tsx

```diff
@@ -1,4 +1,14 @@
-import { getPlanClass, FEATURE_CUSTOM_DOMAIN, isFreePlan } from '@automattic/calypso-products';
+import {
+	getPlanClass,
+	isFreePlan,
+	FEATURE_CUSTOM_DOMAIN,
+	FEATURE_NEWSLETTER_IMPORT_SUBSCRIBERS_FREE,
+	FEATURE_UNLIMITED_SUBSCRIBERS,
+	FEATURE_PAYMENT_TRANSACTION_FEES_10,
+	FEATURE_PAYMENT_TRANSACTION_FEES_8,
+	FEATURE_PAYMENT_TRANSACTION_FEES_4,
+} from '@automattic/calypso-products';
+import { isNewsletterFlow } from '@automattic/onboarding';
```

Comment:
Can we use [PlansIntent](https://github.com/Automattic/wp-calypso/blob/update/newsletter-plans-screen/client/my-sites/plan-features-2023-grid/hooks/npm-ready/data-store/use-grid-plans.tsx#L99) logic for this? It will also sync the changes to both Signup & Admin. You can get the `intent` parameter from context (see uses of `usePlansGridContext`). 

We may consider removing flowName entirely from the grid to rely more on `PlansIntent` - in combination with `isInSignup` (etc.) when needed.

(Also related to my other comment. in case you need to abstract the whole thing out of the grid components)